input/resources/StructureDefinition-profile-patient.json

```diff
@@ -45,6 +45,216 @@
+        "path": "Patient.identifier",
+        "sliceName": "SSN",
+        "short": "SSN Identifier for this patient",
+        "definition": "SSN identifier for this patient."
+				  
+      },
+      {
+        "id": "Patient.identifier:SSN.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:SSN.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "SB"
+      },
+      {
+        "id": "Patient.identifier:SSN.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "SSN"
+      },
+      {
+        "id": "Patient.identifier:SSN.type.text",
+        "path": "Patient.identifier.type.text",
+        "fixedString": "Social Security Number"
+      },
+      {
+        "id": "Patient.identifier:SSN.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "http://hl7.org/fhir/sid/us-ssn"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID",
+        "path": "Patient.identifier",
+        "sliceName": "MedicaidID",
+        "short": "Medicaid identifier for this patient",
+        "definition": "Medicaid identifier for this patient."
+				  
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "Medicaid ID"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.type.text",
+        "path": "Patient.identifier.type.text",
+																																														   
+        "fixedString": "Patient Medicaid number"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "https://signalbhn.org/fhir/sid/us-medicaidid"
+      },
+      {
+        "id": "Patient.identifier:MedicareID",
+        "path": "Patient.identifier",
+        "sliceName": "MedicareID",
+        "short": "Medicare identifier for this patient",
+        "definition": "Medicare identifier for this patient."
+				  
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"

```

Comment:
Was reading this https://jira.hl7.org/browse/FHIR-27538, chaange MedicaidID and Medicare ID:
type.coding.code = SB
type.coding.display = Social Beneficiary Identifier

Reply:
changed